### PR TITLE
Query lastTransactions based on time

### PR DESCRIPTION
**Pull request by vponline**
Return last transaction results for the last 12 hours with a maximum of 720. 

Closes: https://api3dao.atlassian.net/browse/BEC-623

### CODE REVIEW

File: src/last-transactions.ts [hunk 1]
```diff
@@ -96,7 +99,8 @@ export const lastTransactions: APIGatewayProxyHandler = async (event): Promise<A
     FROM dapi_events 
     WHERE chain = $1 AND 
     event_name IN ('UpdatedBeaconWithSignedData', 'UpdatedBeaconWithPsp', 'UpdatedBeaconWithRrp', 'UpdatedBeaconSetWithBeacons', 'UpdatedBeaconSetWithSignedData') AND
-    event_data->'parsedLog'->'args'->> 0 = $2
+    event_data->'parsedLog'->'args'->> 0 = $2 AND
+    time > NOW() - INTERVAL '12 HOURS'
```
Comment (vponline) [inline src/last-transactions.ts:103]: I think 12 hours should be enough with a limit of `MAX_TRANSACTION_COUNT_LIMIT` of 720 (i.e. a max of 1 update per minute for 12 hours).

Comment (aquarat) [reply src/last-transactions.ts:103]: Perfect 💪

**Review by aquarat — CHANGES_REQUESTED** [review]
The transaction count limit query string handling code is still here, is that intentional?
Your update to the SQL query should render that parameter redundant.

**Comment by vponline** [comment]
> The transaction count limit query string handling code is still here, is that intentional?
> Your update to the SQL query should render that parameter redundant.

Removed now :smile:

**Review by aquarat — APPROVED** [review]
LGTM :) 
I'll push the config to Keybase for you asap and let you know once done (thought I had already done so but can't see it).

Update: I prepared the git repo but didn't push it, look for it in api3_market_api and let me know if you don't find it. I note that there is a second deployment currently on the AWS account, so you may need to delete all those functions/the cloud formation and start from fresh.

**Pull request merged**